server/resource_server/repository/repository_test.go

```diff
@@ -1501,3 +1502,68 @@ func TestRegisterUserv2_Success(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Nil(t, mock.ExpectationsWereMet(), "There were unfulfilled expectations!")
 }
+
+func TestUpdateUserPasswordV2_Success(t *testing.T) {
+	SetUp(t)
+	defer mockDB.Close()
+
+	mock.ExpectBegin()
+
+	mock.ExpectExec(
+		regexp.QuoteMeta(`UPDATE "users" SET "server_key"=$1,"stored_key"=$2 WHERE username=$3`),
+	).WithArgs(serverKey, storedKey, username).
+		WillReturnResult(sqlmock.NewResult(1, 1))
+
+	mock.ExpectCommit()
+
+	err := repository.UpdateUserPasswordV2(username, storedKey, serverKey)
```

Comment:
Add a blank line here to separate call of the function under test from expectations of the test function output